src/prompt.js

```diff
@@ -77,52 +49,42 @@ const prompt = async (opts) => {
       filter: (val) => val.toLowerCase(),
     },
     {
-      type: 'confirm',
-      name: 'api',
-      message: 'Use Graasp API',
-      default: true,
+      type: 'list',
+      message: 'Language',
```

Comment:
I would specify "Programming Language" to avoid confusion with natural languages.